flags/eigendaflags/cli.go

```diff
@@ -27,6 +28,8 @@ var (
 	ConfirmationDepthFlagName            = withFlagPrefix("confirmation-depth")
 	EthRPCURLFlagName                    = withFlagPrefix("eth-rpc")
 	SvcManagerAddrFlagName               = withFlagPrefix("svc-manager-addr")
+	// Flags that are proxy specific, and not used by the eigenda-client
+	PutRetriesFlagName = withFlagPrefix("put-retries")
```

Comment:
Maybe... but then it would have its own category, which is kind of weird. If we have more eigenda related stuff I think it makes sense for them to have them all under the same category. We can wait and see if we ever add more flags here and then reconsider the category.